src/bin/rage/main.rs

```diff
@@ -314,7 +314,10 @@ fn main() -> Result<(), error::Error> {
 
     if opts.decrypt {
         decrypt(opts).map_err(error::Error::from)
-    } else {
+    } else if opts.recipient.len() > 0 {
```

Comment:
Instead of triggering on empty recipients, this should trigger on completely empty options. Currently this PR will break password encryption, where `-p` is present and `opts.recipient` is empty.

As a side-note, this would be better rewritten as `else if !opts.recipient.is_empty()`.